addons/website/static/src/components/views/page_list.xml

```diff
@@ -36,4 +39,23 @@
     </div>
 </t>
 
+<t t-name="website.ActiveWebsiteFacet">
+    <div class="o_searchview_facet o_facet_active_website position-relative d-inline-flex align-items-stretch rounded-2 bg-200 text-nowrap opacity-trigger-hover"
```

Comment:
IMO, `o_facet_active_website` no need to introduce new class here. keep it simple & clear.